README.md

```diff
@@ -2,100 +2,128 @@
 
 [![Build Status](https://secure.travis-ci.org/jzaefferer/commitplease.png)](http://travis-ci.org/jzaefferer/commitplease)
 
-This [node.js](http://nodejs.org/) module validates git commit messages while you commit, according to the rules specified in the [jQuery Commit Guidelines](http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines). These are pretty generic, so this module should be useful for various projects that care about commit message formats.
+This [node.js](http://nodejs.org/) module makes sure your commit
+messages consistently follow one of these style guides:
+ 1. [jQuery Commit Guidelines][1]
+ 2. [AngularJS Commit Guidelines][2]
+ 3. Your own (if you really feel like it)
 
 ## Installation
 
 ```js
-npm install commitplease
+npm install commitplease --save-dev
 ```
 
-A git version of 1.8.5 or newer is recommended. If you use `git commit --verbose`, it is required.
+A git version of 1.8.5 or newer is recommended. If you use `git commit
+--verbose`, it is required.
 
 ## Usage
 
-Just commit as usual. This modules installs a git commit-msg hook (unless suppressed, see below), automatically validating all commit messages as you enter them. Invalid messages will be rejected, with details on what's wrong and a copy of the input.
+Just commit as usual. This module is triggered by a git commit-msg
```

Comment:
Let's drop the "just", should avoid that in general, same for "easy" or "simple"